tools/cldr-code/src/main/java/org/unicode/cldr/icu/TestCldr.java

```diff
@@ -223,7 +224,7 @@ void set(AttributeName attributeName, String attributeValue) {
         }
 
         void checkResult(String value) {
-            ULocale ul = new ULocale("xx");
+            ULocale ul = new ULocale(LocaleNames.XX_TEST);
```

Comment:
why XX and not `ZZ`?

Reply:
I have no idea! I don't like to see a mysterious locale ID like "xx" hidden away as a string; defining a constant in LocaleNames.java is a step towards clarifying what it is. If it can be removed, so much the better. I can address that in a follow-up PR.